### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by Kaitul**
Fixed some minor spelling and punctuation issues

### CODE REVIEW

**Comment by bluehomewu** [comment]
> @bluehomewu thoughts?

I think some strings changes are unnecessary.
I'll point out the unnecessary parts later.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -209,11 +209,10 @@
...
     <string name="donations__button_close">關閉</string>
     <string name="donations__google_android_market">Google Play 商店</string>
-    <string name="donations__google_android_market_not_supported">不支援應用程式內購。Play 商店是否正確安裝?</string>
-    <string name="donations__google_android_market_donate_button">抖內！</string>
-    <string name="donations__google_android_market_text">抖內多少錢</string>
-    <string name="donations__thanks_dialog">感謝抖內！\n十分感謝您！</string>
-
-    <string name="donations__description">這個程式有用嗎？\n小額的抖內給開發人員用來支持此應用程式的開發</string>
+    <string name="donations__google_android_market_not_supported">不支援應用程式內捐助，Play 商店是否正確安裝？</string>
+    <string name="donations__google_android_market_donate_button">捐助！</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:213]: This part is an additional supplement to be more friendly to users. No need to update translations.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 2]
```diff
@@ -209,11 +209,10 @@
...
+    <string name="donations__google_android_market_text">捐助多少錢</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:214]: This part is an additional supplement to be more friendly to users. No need to update translations.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 3]
```diff
@@ -209,11 +209,10 @@
...
+    <string name="donations__thanks_dialog">感謝捐助！\n十分感謝您！</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:215]: This part is an additional supplement to be more friendly to users. No need to update translations.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 4]
```diff
@@ -209,11 +209,10 @@
...
+    <string name="donations__description">這個應用程式有用嗎？\n小額的捐助給開發人員用來支援此應用程式的開發</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:216]: This part is an additional supplement to be more friendly to users. No need to update translations.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 5]
```diff
@@ -137,27 +137,27 @@
...
     <string name="settings_misc">雜項</string>
     <string name="settings_help">說明</string>
     <string name="settings_misc_logcat">匯出偵錯資訊</string>
-    <string name="settings_misc_logcat_summary">這種非常有用啊 (^O^)／</string>
+    <string name="settings_misc_logcat_summary">如此有用 (^0^)</string>
     <string name="settings_misc_source">專案首頁 @ GitHub</string>
-    <string name="settings_misc_source_summary">閱讀使用說明，star，提交 issues，合作</string>
-    <string name="settings_misc_donate">抖內</string>
+    <string name="settings_misc_source_summary">閱讀手冊、加星號、提交問題並參與</string>
+    <string name="settings_misc_donate">捐助</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:158]: This part is an additional supplement to be more friendly to users. No need to update translations.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 6]
```diff
@@ -18,7 +18,7 @@
     <string name="title_update">更新</string>
 
     <string name="repeater_channel">中繼器 (%1$d MHz，頻道 %2$d)</string>
-    <string name="repeater_wps">WPS（不安全）</string>
+    <string name="repeater_wps">WPS (不安全)</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:21]: Chinese (Traditional) uses full-width punctuation, so it doesn't have to be changed.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 7]
```diff
@@ -90,9 +90,9 @@
     <string name="tethering_manage_wifi_client_blocked">已隱藏 %1$s：%2$s</string>
     <string name="tethering_manage_wifi_copy_mac">複製 MAC</string>
 
-    <string name="connected_state_incomplete">（正在連線）</string>
-    <string name="connected_state_valid">（已連線）</string>
-    <string name="connected_state_failed">（未連線）</string>
+    <string name="connected_state_incomplete">(正在連線)</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:93]: Chinese (Traditional) uses full-width punctuation, so it doesn't have to be changed.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 8]
```diff
@@ -90,9 +90,9 @@
...
+    <string name="connected_state_valid">(已連線)</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:94]: Chinese (Traditional) uses full-width punctuation, so it doesn't have to be changed.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 9]
```diff
@@ -90,9 +90,9 @@
...
+    <string name="connected_state_failed">(已中斷)</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:95]: Chinese (Traditional) uses full-width punctuation, so it doesn't have to be changed.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 10]
```diff
@@ -137,27 +137,27 @@
...
     <string name="settings_system_tether_offload_summary">系統「開發人員選項」的捷徑</string>
     <string name="settings_misc">雜項</string>
     <string name="settings_help">說明</string>
     <string name="settings_misc_logcat">匯出偵錯資訊</string>
-    <string name="settings_misc_logcat_summary">這種非常有用啊 (^O^)／</string>
+    <string name="settings_misc_logcat_summary">如此有用 (^0^)</string>
     <string name="settings_misc_source">專案首頁 @ GitHub</string>
-    <string name="settings_misc_source_summary">閱讀使用說明，star，提交 issues，合作</string>
-    <string name="settings_misc_donate">抖內</string>
+    <string name="settings_misc_source_summary">閱讀手冊、加星號、提交問題並參與</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:157]: This part is an additional supplement to be more friendly to users.
But I suggest this part translations to
`閱讀使用說明，對 GitHub repo 加星號，提交問題並合作參與開發`

**Review by bluehomewu — CHANGES_REQUESTED** [review]

File: mobile/src/main/res/values-zh-rTW/strings.xml (hunk 6 above)
Comment (bluehomewu) [reply mobile/src/main/res/values-zh-rTW/strings.xml:21]: https://github.com/sparanoid/chinese-copywriting-guidelines#%E4%BD%BF%E7%94%A8%E5%85%A8%E5%BD%A2%E4%B8%AD%E6%96%87%E6%A8%99%E9%BB%9E

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 11]
```diff
@@ -137,27 +137,27 @@
...
     <string name="settings_service_dhcp_workaround">啟用 DHCP 因應措施</string>
-    <string name="settings_service_dhcp_workaround_summary">如果裝置無法取得 IP 位址，嘗試開啟這選項。</string>
+    <string name="settings_service_dhcp_workaround_summary">如果裝置無法取得 IP 位址，嘗試開啟這選項</string>
     <string name="settings_system_tether_offload">網路共用硬體加速</string>
     <string name="settings_system_tether_offload_summary">系統「開發人員選項」的捷徑</string>
     <string name="settings_misc">雜項</string>
     <string name="settings_help">說明</string>
     <string name="settings_misc_logcat">匯出偵錯資訊</string>
-    <string name="settings_misc_logcat_summary">這種非常有用啊 (^O^)／</string>
+    <string name="settings_misc_logcat_summary">如此有用 (^0^)</string>
```
[the remark this answers is not in the document]
Comment (bluehomewu) [reply mobile/src/main/res/values-zh-rTW/strings.xml:155]: I can accept changes to symbols, but not their strings

File: mobile/src/main/res/values-zh-rTW/strings.xml (hunk 6 above)
Comment (Kaitul) [reply mobile/src/main/res/values-zh-rTW/strings.xml:21]: But the fact is, Apple, Google, Microsoft are not using fullwidth brackets in the operating system and software Chinese translation.

Here are some examples copied from their website or app:
* 雲端硬碟、Gmail 和其他產品 (111.5 MB)
* 使用系統選擇 (預設)
* 允許將任何應用程式寫入外部儲存空間 (無論資訊清單值為何)
* Apple 與 Apple 的換購合作夥伴保留以任何理由拒絕或限制接受任何裝置 (及其數量) 參與本優惠。
* 此選項適用於想要建立可開機的安裝媒體 (USB 快閃磁碟機或 DVD) 或建立虛擬機器 (.ISO 檔案) 以安裝 Windows 11 的使用者。此下載是多版本 ISO，其使用您的產品金鑰解除鎖定正確的版本。

**Comment by pan93412** [comment]
@Kaitul There is no correct or wrong in terms of style; however, the consensus is essential: as the earliest translator, Edward has determined the translation style of this software in his translation. Even if his translation style does not match the translation guide of Microsoft or Google, his style should still be respected. Besides, this software is obviously not affiliated with Microsoft, Google, or even Apple – there is no reason to adopt the translation style specifications of these companies completely. Ultimately, if you are really dissatisfied with this style, it must be more appropriate to open an issue for a constructive discussion than change the styles without the original author's authorization.

**Comment by Kaitul** [comment]
> @Kaitul There is no correct or wrong in terms of style; however, the consensus is essential: as the earliest translator, Edward has determined the translation style of this software in his translation. Even if his translation style does not match the translation guide of Microsoft or Google, his style should still be respected.
> 
> Besides, this software is obviously not affiliated with Microsoft, Google, or even Apple – there is no reason to adopt the translation style specifications of these companies completely. Ultimately, if you are really dissatisfied with this style, it must be more appropriate to open an issue for a constructive discussion than change the styles without the original author's authorization.

I'm sorry, I will respect Edward's translation style and revise the existing translation, please let me know if you have any requests.

I have changed some Chinese terms in the last version, this update is only a supplement to the last change, because zh-rTW seems to be a direct translation from zh-rCN.

**Comment by bluehomewu** [comment]
> > @Kaitul There is no correct or wrong in terms of style; however, the consensus is essential: as the earliest translator, Edward has determined the translation style of this software in his translation. Even if his translation style does not match the translation guide of Microsoft or Google, his style should still be respected.
> > Besides, this software is obviously not affiliated with Microsoft, Google, or even Apple – there is no reason to adopt the translation style specifications of these companies completely. Ultimately, if you are really dissatisfied with this style, it must be more appropriate to open an issue for a constructive discussion than change the styles without the original author's authorization.
> 
> I'm sorry, I will respect Edward's translation style and revise the existing translation, please let me know if you have any requests.
> 
> I have changed some Chinese terms in the last version, this update is only a supplement to the last change, because zh-rTW seems to be a direct translation from zh-rCN.

I used English to translate, just because I noticed that the original author(@Mygod) maybe Chinese ( I'm not sure, just because he upload zh-rCN strings, so I guess) .
Therefore, some translations refer to the original author's own translation.Respect what the original author wanted to express

**Comment by Kaitul** [comment]
> author

Sorry, I didn't think of that. I only have my phone in my hand now, I'll continue editing when I can, it's bedtime now, good night! :)

File: mobile/src/main/res/values-zh-rTW/strings.xml (hunk 11 above)
[the remark this answers is not in the document]
Comment (Brli) [reply mobile/src/main/res/values-zh-rTW/strings.xml:155]: Check that the original emojis of JP AA or TW BBS culture, where they mostly use full width

[the remark this answers is not in the document]
Comment (Mygod) [reply mobile/src/main/res/values-zh-rTW/strings.xml:155]: Personally I do prefer the previous version. 🙃

**Comment by Mygod** [comment]
I agree with @pan93412. Let's stick with what native speakers of zh-rTW (e.g. @bluehomewu) suggest. Aesthetic styling is not an issue as long as it is consistent within the app.

**Comment by bluehomewu** [comment]
> I agree with @pan93412. Let's stick with what native speakers of zh-rTW (e.g. @bluehomewu) suggest. Aesthetic styling is not an issue as long as it is consistent within the app.

Thanks>👍

**Review by bluehomewu — APPROVED** [review]

**Comment by Mygod** [comment]
Merged. Thanks everyone!

**Pull request merged**